Simplicity.C.nix

```diff
@@ -1,13 +1,22 @@
-{ lib, stdenv, wideMultiply ? null }:
+{ lib, stdenv, gcovr ? null, wideMultiply ? null, withCoverage ? false
+, gcov-executable ? "gcov" # Set to "${nixpkgs.libllvm}/bin/llvm-cov gcov" when using clang.
+}:
 assert wideMultiply == null
     || wideMultiply == "int64"
     || wideMultiply == "int128"
     || wideMultiply == "int128_struct";
+assert withCoverage -> gcovr != null;
```

Comment:
...and I'm confused as to how I avoided triggering this assertion in my code.

Reply:
gcovr is part part of nixpkgs and so it is automatically included.

Maybe I'll just go ahead and delete this assertion.  It is a bit silly.